content/post/career-ladders-part-2.md

```diff
@@ -0,0 +1,56 @@
+
+## Guiding Principles
+Guiding principles are simple heuristics that capture how roles differ from each other. You don’t get promoted for just getting better at the same job, you get promoted for showing that you can do a different job.
+
+Some examples of guiding principles:
+- A general management heuristic we use at [Yerdle](https://www.yerdlerecommerce.com/about-us): a Manager executes a program with some direction, while a Director can create a program from scratch.
+- Software Engineering at [Microsoft](https://www.quora.com/What-are-the-differences-between-SDE-SDE-II-and-Senior-SDE-at-Microsoft): a Software Engineer creates complex solutions to simple problems, a Software Engineer II creates simple solutions to simple problems, a Senior Software Engineer creates simple solutions to complex problems, and a Principal Software Engineer makes complex problems disappear.
+- Design at [Lyst](https://medium.com/designing-lyst/career-progression-for-product-designers-user-researchers-at-lyst-49c0c6cb9003): Junior contributes to the work, Mid does the work, Senior leads the work, and Lead leads the team.
+- Data Science, according to [Jan Zawadzki](https://towardsdatascience.com/becoming-a-level-3-0-data-scientist-52641ff73cb3): Level 1 data scientsts have strong statistics skills, Level 2 adds engineering skills, and Level 3 adds business skills.
+
+My own principles are not so pithy. When I think about how analytics team members work at different levels and how to clearly articulate the difference between roles, there are two key areas where I see step changes with experience: scope of influence, and what deliverables look like. Here is my matrix for those principles for individual contributors:
+
+{{< figure src="/img/career-ladders-guiding-principles.png" title="Guiding principles for individual contributors">}}
+
+Guiding principles set the stage for the specific competencies that come next. But importantly, they also enable you to create flexibility and variations of the ladder as needed. You can modify a career ladder for various flavors - if your team includes data scientists, analytics engineers, data product managers, data engineers, or other roles, you can outline guiding principles that work across disciplines, providing continuity in levels as well as making the path between specialties clearer. You can even modify a career ladder for a single individual who wants to follow a specialist path if that makes sense for the team. I manage fairly small teams and am happy to accommodate different career paths; knowing the guiding principles has allowed me to create a Product/UX Analyst ladder and a Sales Operations Analyst ladder that spoke to the skills and roles of particular individuals more specifically, but still aligned very clearly with the team's overall leveling framework.
```

Comment:
Is it fair to say that guiding principles apply to all roles, "across the team", vs. specific competencies, which are role specific?